src/hotspot/share/gc/shenandoah/mode/shenandoahMode.hpp

```diff
@@ -25,6 +25,7 @@
 #ifndef SHARE_GC_SHENANDOAH_MODE_SHENANDOAHMODE_HPP
 #define SHARE_GC_SHENANDOAH_MODE_SHENANDOAHMODE_HPP
 
+#include "runtime/java.hpp"
```

Comment:
This include looks unused.

Reply:
Moved to "shenandoahGenerationalMode.cpp".